Expensify/App
# Sidebar UI - Consumer UI Refresh

**Details**
For this issue we must change the icon of the "hamburger" menu and do some light refactoring and styling.

- Remove all language in the code that refers to hamburger wherever hamburger is used we should replace it to navigationMenu

- Remove the hamburger menu icon itself from the HeaderView and replace it with this chevron-left back style icon. Switch it to a Pressable instead of a TouchableOpacity.

- Make the "toggled open" state of the menu take up the entire width of the screen when viewing below the breakpoint.
- Remove the Pressable that is listening for touches outside of the nav since the nav is now either pinned or full screen there is no need to listen for touches on an area that doesn’t exist.


**Fixed Issues**
https://github.com/Expensify/Expensify.cash/issues/1223

**Tests**

1. login with the app with credential. 
2. Check the main page, hamburger replace with back. 
3. Click on back, Open the side menu. it will be full screen now
4. Code change hamburger with navigationMenu, it will be in code.


**Tested On**


**Screenshots**


#### Web


#### Mobile Web


#### Desktop


#### iOS
<img width="465" alt="Screenshot 2021-01-14 at 12 14 53 AM" src="https://user-images.githubusercontent.com/10494663/104631917-f2acff00-56c2-11eb-9266-f633a9c9606b.png">

![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 23 00 11](https://user-images.githubusercontent.com/10494663/104631944-fccefd80-56c2-11eb-862c-e265a208f08d.png)


#### Android
